src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -29,6 +34,11 @@ public void promptNextPlayer(){
         }
     }
 
+    public void printScore(){
+        System.out.println("X wins: " + score[0]  + "    O wins: " + score[1] +
+                    "    Ties:" + score[2]+ "");
+    }
```

Comment:
The feature 2 requirement shows to keep track of how many games were won, lost, tied by each player.
Therefore, you should add a lost score.